docs/impact/index.md

```diff
@@ -144,25 +117,32 @@ github-dependents-info --repo bids-standard/pybids --badgemarkdownfile ./dashboa
 | ---------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | pybids     | [![](https://img.shields.io/static/v1?label=Used%20by&message=596&color=informational&logo=slickpic)](https://github.com/bids-standard/pybids/network/dependents) |
 
-## Measuring BIDS Impact
+## BIDS History
 
-### Impact summary
+Since its origin, the BIDS standard has revolutionized the way in which neuroimaging
+research is done. The origin of the BIDS standard is commonly traced back to 2014, 
+when a social media post ignited a deeper discussion about a new way of sharing 
+data that would ultimately lead to a set of standards that would improve scientific 
+collaboration efforts. 
 
-Since its origin, the BIDS has revolutionized the way in which neuroimaging
-research is done.
+Following this initial discussion, and on the shoulders of many many volunteer hours 
+by maintainers and the community at large, the BIDS standard has seen significant adoption 
+in the neuroimaging research landscape and continues to expand. A more complete history 
+and discussion about the future of the BIDS landscape can be found in this 
+[recent publication](https://pmc.ncbi.nlm.nih.gov/articles/PMC10516110/).
 
-~??? users visiting the website, and ~??? users exploring the BIDS
-Specification, over the past ??? months.
+The BIDS standard continues to be adopted by a larger share of the community each year, 
+and we have had ~??? users visiting the BIDS website and ~??? users exploring the BIDS
+Specification over the past ??? months.
```

Comment:
@ericearl did this get lost to time in google analytics?

Reply:
it looks like the info is still being collected, but I went to the dashboard and it looks like I don't have permission to view. I initially tried to pull from there. Can someone look into perms? If we're linking it on the site it should also be open to everyone, right?

Reply:
Yeah, I would comment out the dashboard for analytics if it's not going to load. I don't have permissions either so all I see if    bunch of boxes with `No Data Set Access`.

Reply:
who can help, @Remi-Gau ?
What's the best way to get unstuck?

Reply:
I checked today and the Google Analytics on the BIDS Spec don't appear to have been collected correctly since the release of BIDS 1.10.0. I propose we strike this question-mark-filled sentence and ask the BIDS Maintainers to come back and fill this in later.

Alternatively, after GSOD ends this year, we can ask the Online Presence Working Group to come up with PRs for incremental Impact page improvements.

Reply:
I would say that fixing google analytics is beyond the scope of this.

If we have some dead or useless links, let's comment out the sections that include them, open an issue and fix later.